dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/enrollment/AbstractEnrollmentService.java

```diff
@@ -562,18 +641,11 @@ private ImportSummary addEnrollment( Enrollment enrollment, ImportOptions import
         enrollment.setEnrollment( programInstance.getUid() );
         importSummary.getImportCount().incrementImported();
 
-        if ( handleEvents )
-        {
-            importSummary.setEvents( handleEvents( enrollment, programInstance, importOptions ) );
```

Comment:
not sure but this flag was always false. Events are persisted later on

Reply:
I thought they were persisted here? At least thats how it worked before,  the entire import is nested. Remember there is a difference between event from event programs and from tracker program; They are imported differently, since event program events are not nested within an enrollment for example.